project/plugins.sbt

```diff
@@ -1,2 +1,2 @@
 addSbtPlugin("com.github.sbt" % "sbt-dynver" % "5.1.0")
-addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.9.3")
+addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.11.0")
```

Comment:
I'm in the process of migrating com.github.sbt to the Central Portal. Please update to sbt-ci-release 1.11.0 only after that's done.